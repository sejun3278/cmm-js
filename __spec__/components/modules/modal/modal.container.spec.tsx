import React from "react";

import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import ModalPage from "../../../../src/components/modules/modal/modal.container";
// import ModalPage from "../../../../src/components/modules/test";

// @ts-ignore
export default it("Modal Page Sanpshot", () => {
  const component = render(
    <ModalPage
      //   str1="1"
      //   str2="2"
      //   onModalOpenAnimation={() => {}}
      show={true}
      onCloseModal={() => {}}
      onBGAnimation={false}
      onModalOpenAnimation={false}
    />
  );
  // @ts-ignore
  expect(component.container).toMatchSnapshot();
});
