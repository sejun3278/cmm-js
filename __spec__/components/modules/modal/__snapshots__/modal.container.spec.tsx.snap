// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Page Sanpshot 1`] = `
<div>
  <div
    class="cmm-modal-wrapper css-7jraf2"
  >
    <div
      class="cmm-modal-item css-1dgb740"
    >
      <div
        class="cmm-modal-close-wrapper css-1d7mo96"
      >
        <button
          class="cmm-modal-close-button css-el5ojm"
        >
          <img
            class="cmm-unit-image cmm-modal-close-img css-msby2z"
            src="https://iconmonstr.com/wp-content/g/gd/makefg.php?i=../releases/preview/7.2.0/png/iconmonstr-x-mark-lined.png&r=0&g=0&b=0"
          />
        </button>
      </div>
      <div
        class="cmm-modal-contents-wrapper css-d4a1oh"
      >
        <div
          class="cmm-modal-content css-3o9g8b"
        />
      </div>
    </div>
  </div>
</div>
`;
